src/main/java/com/thed/service/impl/ZephyrRestServiceImpl.java

```diff
@@ -139,6 +139,7 @@ public Boolean login(String hostAddress, String username, String password) throw
         httpClientService.clear();
         String encoding = Base64.getEncoder().encodeToString((username+":"+password).getBytes());
         httpClientService.getHeaders().add(new BasicHeader("Authorization", "Basic "+encoding));
+        httpClientService.getHeaders().add(new BasicHeader("accessType", "HTMLUI"));
```

Comment:
@KrzysiekAgi Would sending this accessType header logout existing UI session?

Reply:
@pv-smartbear It appears from our tests that it will. Is this a big problem? Users should be using a technical account for communicating with zephyr via API, right?

Reply:
@KrzysiekAgi Let's discuss this over a call.

Reply:
As discussed we will send a Basic Auth header with each API request. This will keep the users from logging out and maintain the current behavior.